avm/res/compute/virtual-machine/main.bicep

```diff
@@ -453,6 +453,7 @@ module vm_nic 'modules/nic-configuration.bicep' = [
 resource vm 'Microsoft.Compute/virtualMachines@2022-11-01' = {
   name: name
   location: location
+  #disable-next-line BCP036
```

Comment:
Trying to fix this later. (this is already in main @rahalan )

Reply:
@johnlokerse, not sure I understand the comment? Is there something I can help?

Reply:
Sorry for the confusion. I came across this error which was already in the main branch and it was a reminder for me to try and fix it. Just tagged you for visibility :-)

Reply:
When disabling I'd in general recommend to add a comment. These identifiers are rather cryptic after all 😄

Reply:
Fixed it, was a small typo in the variable. A "space" was missing:

<img width="1032" alt="image" src="https://github.com/Azure/bicep-registry-modules/assets/3514513/ac11ba65-93df-453a-a9ed-703c411f02fa">